start.sh

```diff
@@ -0,0 +1,9 @@
+DIR="data"
+
+if [ -d "$DIR" ]; then
+    rm -rf $DIR
+fi
+
+mkdir $DIR
+
+docker run --rm --name=scraper -v $PWD/data:/app/scraper/data liquidinvestigations/crji-avere-europarl python3 /app/scraper/main.py --id 124831
```

Comment:
Remove the `--id` argument here, since this is the documented way for users to run this.